tests/test_library_commands.py

```
@@ -0,0 +1,373 @@
+
+
+class TestLibraryInstall:
+    """Tests for library install command."""
+
+    def _mock_fetch_index(self, *args, **kwargs):
+        return IndexData.from_dict(SAMPLE_INDEX), False
+
+    def _mock_fetch_evaluator(self, provider, name):
+        return SAMPLE_EVALUATOR_YAML
+
+    def test_install_success(self, capsys):
```

Ruff (0.14.14)

[warning] Unused method argument: `capsys`

(ARG002)